third_party/move/move-model/src/exp_rewriter.rs

```diff
@@ -9,15 +9,15 @@ use crate::{
     ty::Type,
 };
 use itertools::Itertools;
-use std::collections::{BTreeSet, VecDeque};
+use std::collections::BTreeSet;
 
 /// Rewriter for expressions, allowing to substitute locals by expressions as well as instantiate
 /// types.
 pub struct ExpRewriter<'env, 'rewriter> {
     env: &'env GlobalEnv,
     replacer: &'rewriter mut dyn FnMut(NodeId, RewriteTarget) -> Option<Exp>,
     type_args: &'rewriter [Type],
-    shadowed: VecDeque<BTreeSet<Symbol>>,
+    shadowed: Vec<BTreeSet<Symbol>>,
```

Comment:
Why this change?

Reply:
A `VecDeque` is strictly more powerful/complicated than a `Vec`, and we're not using any of that.  So it should be a `Vec`.